jekyll/_cci2/pipelines.adoc

```diff
@@ -94,14 +112,16 @@ To add a trigger for a pipeline, follow these steps:
 
 Setting up a custom webhook trigger allows you to trigger pipelines from external services. Any service that can emit a webhook or make a cURL request can be set up in this way to trigger a pipeline. For more information, see the xref:triggers-overview#trigger-a-pipeline-from-a-custom-webhook[Triggers overview] page.
 
+NOTE: Custom webhooks are available for GitHub App pipelines only. This includes GitHub App pipelines set up within a xref:github-integration#using-github-app-functionality[GitHub OAuth org].
```

Comment:
Can we suggest that users look at their Project Settings > Pipelines page to check if they have a pipeline with the "GitHub App" pill on it, and if not they create one?

Reply:
Also I'm reading this again, I think "are available" makes it sound like certain segments of users can't access them. 
How about `Custom webhook triggers can only be added to (or associated to) GitHub App pipelines. You can check if you have a GitHub App pipeline by going to Project Settings > Pipelines and checking if any of your pipelines has a "GitHub App" pill on the Config Source column. If needed, create a new pipeline through the button "Add pipeline".`